WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -119,7 +119,18 @@ private async Task<InputRegistrationResponse> RegisterInputCoreAsync(InputRegist
 			alice.SetDeadlineRelativeTo(round.ConnectionConfirmationTimeFrame.Duration);
 			round.Alices.Add(alice);
 
-			CoinVerifier?.TryScheduleVerification(coin, round.InputRegistrationTimeFrame.EndTime, out _, cancellationToken, oneHop, confirmations);
+			int currentBlockchainHeight = 0;
+
+			try
+			{
+				currentBlockchainHeight = await Rpc.GetBlockCountAsync(cancellationToken).ConfigureAwait(false);
+			}
+			catch (Exception ex)
+			{
+				Logger.LogError($"Couldn't get current blockchain height. Exception: {ex}");
+			}
```

Comment:
I hate this, but in tests, this throws an exception and input will be rejected.
So I wrapped around a try-catch block.

I will try to mock the tests, so we don't need the try catch.